basil.js

```diff
@@ -1844,14 +1844,25 @@ pub.previousPage = function () {
 
 
 /**
- * The number of all pages in the current document.
+ * Returns the number of all pages in the current document. If a number is given as an argument,
+ * it will set the document's page count to the given number by either adding pages or removing
+ * pages until the number is reached. If pages are added, the master page of document's last
+ * page will be applied to the new pages.
  *
  * @cat Document
  * @subcat Page
  * @method pageCount
+ * @param  {Number} [pageCount] New page count of the document (integer between 1 and 9999).
  * @return {Number} The amount of pages.
  */
-pub.pageCount = function() {
+pub.pageCount = function(pageCount) {
+  if(arguments.length) {
+    if(pageCount.constructor.name === "Number" && pageCount % 1 === 0 && pageCount > 0 && pageCount < 10000) {
```

Comment:
Looks great. I just needed to look up what  `pageCount % 1 === 0` does. I guess this is a thing we should not forget to do. Write comments into our code to allow future maintainers to not have to google for [sto questions](https://stackoverflow.com/questions/3885817/how-do-i-check-that-a-number-is-float-or-integer).  😄

Reply:
Damn, how did you find my secret source. 😆